src/service/fileMaker.ts

```diff
@@ -4,20 +4,69 @@ import * as fs from '../util/fs';
 import { log } from '../util/logger';
 import { interpolate } from '../util/string';
 import { getTimestampString } from '../util/ts';
+import MakeOptions from '../domain/MakeOptions';
 import Configuration from '../domain/Configuration';
 import FileExtensions from '../enum/FileExtensions';
 import { getMigrationPath } from '../migration/service/knexMigrator';
 
 const MIGRATION_TEMPLATE_PATH = path.resolve(__dirname, '../../assets/templates/migration');
 const CREATE_TABLE_CONVENTION = /create_(\w+)_table/;
+const SOURCE_TYPE_STUBS = {
+  sql: ['create_up.stub', 'create_down.stub'],
+  javascript: ['create_js.stub', 'update_js.stub'],
+  typescript: ['create_ts.stub', 'update_ts.stub']
+};
+
+/**
+ * Get template stub directory path.
```

Comment:
Copy stub files to user directory and return list of template names.

Reply:
Update the doc block.

Reply:
Updated.